aiogram/webhook/aiohttp_server.py

```diff
@@ -90,6 +90,7 @@ def __init__(
         self,
         dispatcher: Dispatcher,
         handle_in_background: bool = False,
+        secret_token: Optional[str] = None,
```

Comment:
This argument is really needed in base request handler?

Is unsuded in base class but only needed in simple reuqest handler.